recipes/google-api-python-client/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "google-api-python-client" %}
+{%set version = "1.5.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "2fd69ea71497a7de01cee4c85a9a64a593458eba0c1fec8d8a24e34d1298fdbd" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - httplib2 >=0.8,<1
+    - oauth2client
+    - six >=1.6.1,<2
+    - uritemplate >=0.6,<1
+
+test:
+  imports:
+    - apiclient
+    - googleapiclient
+    - googleapiclient.discovery_cache
+
+about:
+  home: http://github.com/google/google-api-python-client/
+  license: Apache 2.0 and MIT
```

Comment:
Good eye. Seems there is some mixture. Raised issue ( https://github.com/google/google-api-python-client/issues/252 ) for clarification.